htdocs/api/index.php

```diff
@@ -310,9 +311,22 @@
 // Call API (we suppose we found it).
 // The handle will use the file api/temp/routes.php to get data to run the API. If the file exists and the entry for API is not found, it will return 404.
 
-//Luracast\Restler\Defaults::$returnResponse = true;
+Luracast\Restler\Defaults::$returnResponse = true;
```

Comment:
I suggest to introduce here (and later into section that compress result) an option to be able to disable the compression (we have trouble in past with it).
So to add a 
if (empty($conf->global->API_DISABLE_COMPRESSION)) {

So if user has a trouble with this new native compression enabled by default, we have a quick setup to restore old behaviour.

Reply:
@eldy I've added management of API_DISABLE_COMPRESSION const to be able to return to previous behavior